src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,41 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    public static final int MAX_SIZE = 12;
+    public static final int TOT = -1;
+    public static final int EMPTY = -1;
+    final private int[] numbers = new int[MAX_SIZE];
+
+    public int getTotal() {
+        return total;
+    }
+
+    private int total = EMPTY;
+
+    protected void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+    protected boolean callCheck() {
+        return total == EMPTY;
+    }
+
+    protected boolean isFull() {
+        return total == MAX_SIZE - 1;
+    }
+
+    final protected int peekaboo() {
+        if (callCheck())
+            return EMPTY;
+        return numbers[total];
+    }
+
+    protected int countOut() {
+        if (callCheck())
+            return EMPTY;
+        return numbers[total--];
+    }
```

Comment:
Visibility modifiers should be changed only for class fields. Classes without any visible methods and fields are useless.